src/activ/problem_description.cpp

```diff
@@ -60,6 +60,8 @@ NetworkConfig ProblemDescription::MakeNetworkConfig() const
     std::ostringstream ss;
 
     ss << "activ-";
+    if(direction == Direction::Backward)
```

Comment:
Can we add the else branch too? That would make parsing these easier

Reply:
Sure!